applications/luci-app-opkg/htdocs/luci-static/resources/view/opkg.js

```diff
@@ -662,7 +662,7 @@ function handleInstall(ev)
 		});
 
 	inst = E('p', {},
-		_('Require approx. %.1024mB size for %d package(s) to install.')
+		_('Require approx. %1024mB size for %d package(s) to install.')
```

Comment:
Should it not be `MiB`? Also would it not be better then to convert to Mb instead?